salt/dex/dex.yaml

```diff
@@ -73,6 +73,7 @@ data:
       redirectURIs:
       - 'http://127.0.0.1'
       - 'https://{{ pillar['dashboard'] }}/oidc/done'
```

Comment:
Do we want to keep this redirection? We'll use the new redirection host always, no?

Reply:
As I understand it, we'll use whatever URL the user has in their browser - considering both could be used, I left it in.. I don't *think* it would cause any issues.

Reply:
Kiall's right here - anything that could possibly be used as the redirect has to be here, or Dex will reject it.